tests/cypress/integration/actions_tasks_objects/case_50_settings_player_speed.js

```diff
@@ -18,7 +18,7 @@ context('Settings. "Player speed" option.', () => {
     function changePlayerSpeed(speed) {
         cy.openSettings();
         cy.get('.cvat-player-settings-speed').within(() => {
-            cy.get('.cvat-player-settings-speed-select').click();
+            cy.get('.cvat-player-settings-speed-select').click().wait(300); // Wait for the dropdown menu transition.
```

Comment:
Doesn't the next cy.get() wait automatically?

Reply:
css class ```.cvat-player-settings-speed-<selector>``` appears during the transition dropdown menu. And sometimes at this time, the speed selection string may have the css property ```pointer-events: none``` inherited from the dropdown menu. And at this time, the click may not work. I think that's the reason.